lib/cluster.js

```diff
@@ -33,8 +33,8 @@ function Worker(options) {
 
   if (options.process) {
     this.process = options.process;
-    this.process.on('error', this.emit.bind(this, 'error'));
-    this.process.on('message', this.emit.bind(this, 'message'));
+    this.process.on('error', () => this.emit('error'));
+    this.process.on('message', () => this.emit('message'));
```

Comment:
needs the `message` and `handle` args: https://nodejs.org/api/process.html#process_event_message